spec/unit/target/pod_target_spec.rb

```diff
@@ -842,7 +842,8 @@ module Pod
         it 'returns the correct resource paths' do
           @watermelon_pod_target.resource_paths.should == {
             'WatermelonLib' => [],
-            'WatermelonLib/Tests' => ['${PODS_CONFIGURATION_BUILD_DIR}/WatermelonLibTestResources.bundle'],
+            'WatermelonLib/Tests' => ['${PODS_ROOT}/../../spec/fixtures/watermelon-lib/App/resource.txt',
```

Comment:
This is now correct. test specs should have `test_spec.resources` returned for them because they dont use "Copy Bundle Resources" phase of Xcode directly like app specs do.